### PR TITLE
enable/disable for BgpPeer

### PR DESCRIPTION
**Pull request by drebes**
Partially supersedes GoogleCloudPlatform/magic-modules#3258

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [X] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
compute: Added `enable` attribute to `google_compute_router_peer`
```

### CODE REVIEW

File: mmv1/products/compute/terraform.yaml [hunk 1]
```diff
@@ -2214,6 +2221,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         description: |
           {{description}}
           If it is not provided, the provider region is used.
+      enable: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_expand: 'templates/terraform/custom_expand/bool_to_upper_string.erb'
```
Comment (drebes) [inline mmv1/products/compute/terraform.yaml:2225]: The API attribute is actually an enum supporting only "TRUE" and "FALSE" hence this hack here. This has been discussed in the initial PR this one is based on and we [agreed these was the best way to proceed](https://github.com/GoogleCloudPlatform/magic-modules/pull/3258#discussion_r393939239).

File: mmv1/third_party/terraform/tests/resource_compute_router_bgp_peer_test.go [hunk 2]
```diff
@@ -195,6 +238,7 @@ func testAccComputeRouterPeerBasic(routerName string) string {
 	return fmt.Sprintf(`
 resource "google_compute_network" "foobar" {
   name = "%s-net"
+  auto_create_subnetworks = false
```
Comment (drebes) [inline mmv1/third_party/terraform/tests/resource_compute_router_bgp_peer_test.go:241]: All these examples actually create a subnet, so we disable auto subnet creation to speed things up and not face subnet quota issues.

**Review by ScottSuarez — APPROVED** [review]

**Pull request merged**